drivers/power/pm_initialize.c

```diff
@@ -28,12 +28,6 @@
 
 #include "pm.h"
 
-#if defined(CONFIG_PM_GOVERNOR_ACTIVITY)
```

Comment:
Why this is removed? I see that `activity_governor.h` is only included from `drivers/power/pm_initialize.c` while `greedy_governor.h` is included from `drivers/power/greedy_governor.c` also, but after this change `greedy_governor.h` will become an "empty" include file, so both include files can be removed?

Reply:
> Why this is removed?

Since pm_greedy_governor_initialize and pm_activity_governor_initialize is moved to include/nuttx/power/pm.h in #5701, it is enough to only include nuttx/power/pm.h to reference these functions.

> I see that `activity_governor.h` is only included from `drivers/power/pm_initialize.c` while `greedy_governor.h` is included from `drivers/power/greedy_governor.c` also, but after this change `greedy_governor.h` will become an "empty" include file, so both include files can be removed?

Yes, please review the new update.